lib/api/BackbeatAPI.js

```diff
@@ -736,6 +736,17 @@ class BackbeatAPI {
         }
         try {
             reqBody = JSON.parse(body);
+            // prioritize cancel request
+            if (Object.keys(reqBody).includes('cancel')) {
+                if (reqBody.cancel !== true) {
+                    return {
+                        error: errors.MalformedPOSTrequest.customizeDescription(
+                            `${msg}: cancel key found with incorrect value`),
```

Comment:
I think it would be worthwhile to include in the message that the `cancel` property key must be `true`.